src/homework05/TicTacToe.java

```diff
@@ -0,0 +1,226 @@
+package homework05;
+
+import java.util.Scanner;
+
+public class TicTacToe {
+
+    static String[][] xo = new String[3][3];
+
+    static {
+        for (int i = 0; i < xo.length; i++) {
```

Comment:
pay attention to IDE hints. It recommends to use:
    static {
        for (String[] strings : xo) {
            Arrays.fill(strings, "_");
        }
    }